install_config/http_proxies.adoc

```diff
@@ -23,12 +23,18 @@ files.
 
 . Add the `*NO_PROXY*`, `*HTTP_PROXY*`, and `*HTTPS_PROXY*` environment variables
 ifdef::openshift-origin[]
-to each host's *_/etc/sysconfig/origin-master_* or
-*_/etc/sysconfig/origin-node_* file depending on the type of host:
+to each host's *_/etc/sysconfig/origin-master_* file (for single master
+configuration), or *_/etc/sysconfig/origin-master-api_* and
+*_/etc/sysconfig/origin-master-controllers_* files (for multi-master configuration),
+or *_/etc/sysconfig/origin-node_* file (for node configuration) depending on
+the type of host:
```

Comment:
The structure is basically:

    ...to each host's FILENAME (CONTEXT),
    or FILENAME (CONTEXT),
    or FILENAME and FILENAME (CONTEXT)
    depending on...

I would remove the first "or" and add a comma after the last "(CONTEXT)":

    ...to each host's FILENAME (CONTEXT),
    FILENAME (CONTEXT),
    or FILENAME and FILENAME (CONTEXT),
    depending on...